pkgs/by-name/ve/verovio/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  cmake,
+  git,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "verovio";
+  version = "4.2.0";
+
+  src = fetchFromGitHub {
+    owner = "rism-digital";
+    repo = finalAttrs.pname;
+    rev = "version-${finalAttrs.version}";
+    hash = "sha256-PjBtnt46pHV3rDlwltVxfUMlgNoZGnI9CfnFtk2/j3s=";
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    git
+    makeWrapper
+  ];
+
+  dontConfigure = true;
+
+  buildPhase = ''
+    runHook preBuild
+    set -exo pipefail
```

Comment:
I don't think this is necessary

Reply:
yes, I think this was left over from testing. Removing now.